gpflow/kernels/__init__.py

```diff
@@ -82,6 +82,7 @@
     "Sum",
     "White",
     "base",
+    "categorical",
```

Comment:
Shouldn't this be `Categorical`?

Reply:
No, this is picking up the file `categorical` which includes the class `Categorical`, rather than referring to the class directly.